platform/os/process-kill-parallel_test.go

```diff
@@ -0,0 +1,88 @@
+	"github.com/litmuschaos/litmus-e2e/pkg/log"
+	"github.com/litmuschaos/litmus-e2e/pkg/types"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"k8s.io/klog"
+)
+
+func TestGoProcessKillParallel(t *testing.T) {
+	RegisterFailHandler(Fail)
+	RunSpecs(t, "BDD test")
+}
+
+//BDD for testing experiment
+var _ = Describe("BDD of process-kill experiment", func() {
+
+	// BDD TEST CASE 1 - process-kill in parallel mode
+	Context("Check for process-kill experiment", func() {
+
+		It("Should check for the process kill in parallel", func() {
+
+			testsDetails := types.TestDetails{}
+			clients := environment.ClientSets{}
+			chaosExperiment := v1alpha1.ChaosExperiment{}
+			chaosEngine := v1alpha1.ChaosEngine{}
+
+			//Getting kubeConfig and Generate ClientSets
+			By("[PreChaos]: Getting kubeconfig and generate clientset")
+			err := clients.GenerateClientSetFromKubeConfig()
+			Expect(err).To(BeNil(), "Unable to Get the kubeconfig, due to {%v}", err)
+
+			//Fetching all the default ENV
+			By("[PreChaos]: Fetching all default ENVs")
+			klog.Infof("[PreReq]: Getting the ENVs for the %v test", testsDetails.ExperimentName)
+			environment.GetENV(&testsDetails, "process-kill", "process-kill-engine-par")
+			log.Infof("[Info]: The target processes are: %v", testsDetails.ProcessIds)
+
+			testsDetails.RbacPath = "https://raw.githubusercontent.com/litmuschaos/chaos-charts/m-agent/charts/os/process-kill/rbac.yaml"
+			testsDetails.ExperimentPath = "https://raw.githubusercontent.com/litmuschaos/chaos-charts/m-agent/charts/os/process-kill/experiment.yaml"
+			testsDetails.EnginePath = "https://raw.githubusercontent.com/litmuschaos/chaos-charts/m-agent/charts/os/process-kill/engine.yaml"
+			testsDetails.ChaosNamespace = "default"
+			testsDetails.AppNS = "default"
+
+			// Checking the chaos operator running status
+			By("[Status]: Checking chaos operator status")
+			err = pkg.OperatorStatusCheck(&testsDetails, clients)
+			Expect(err).To(BeNil(), "Operator status check failed, due to {%v}", err)
+
+			// Prepare Chaos Execution
+			By("[Prepare]: Prepare Chaos Execution")
+			err = pkg.PrepareChaos(&testsDetails, &chaosExperiment, &chaosEngine, clients, false)
+			Expect(err).To(BeNil(), "fail to prepare chaos, due to {%v}", err)
+
+			//Checking runner pod running state
+			By("[Status]: Runner pod running status check")
+			err = pkg.RunnerPodStatus(&testsDetails, testsDetails.AppNS, clients)
+			Expect(err).To(BeNil(), "Runner pod status check failed, due to {%v}", err)
+
+			//Chaos pod running status check
+			err = pkg.ChaosPodStatus(&testsDetails, clients)
+			Expect(err).To(BeNil(), "Chaos pod status check failed, due to {%v}", err)
+
+			//Waiting for chaos pod to get completed
+			//And Print the logs of the chaos pod
+			By("[Status]: Wait for chaos pod completion and then print logs")
+			err = pkg.ChaosPodLogs(&testsDetails, clients)
+			Expect(err).To(BeNil(), "Fail to get the experiment chaos pod logs, due to {%v}", err)
```

Comment:
```suggestion
			Expect(err).To(BeNil(), "failed to get the experiment chaos pod logs, due to {%v}", err)
```

Reply:
Can you make this change at all similar places

Reply:
We have used the same format in all the BDDs, we should update all of them if we are going to change it here. However, I don't feel that the current statements are necessarily wrong so we can leave them as it is. Or else, you can create an issue so that we can take it as a common task across all the BDDs. CC: @uditgaurav